demo/config/alertmanager/alertmanager.yml

```diff
@@ -0,0 +1,14 @@
+
+route:
+  receiver: 'mail'
+  repeat_interval: 4h
+  group_by: [ alertname ]
+
+
+receivers:
+  - name: 'mail'
+    email_configs:
+      - smarthost: 'smtp.freesmtpservers.com:25'
+        from: 'alertmanager@ef81f8d5-ba72-4da8-8e2b-1e7525214ec2.com'
```

Comment:
Is this something that anyone could access? I wonder whether we could better do this with a docker container with an smtp server?